app/.well-known/nostr.json/route.ts

```diff
@@ -0,0 +1,45 @@
+import { NextRequest } from "next/server"
+import { promises as fs } from 'fs';
+
+// NIP-05 support
+// See https://github.com/nostr-protocol/nips/blob/master/05.md#example
+export async function GET(request: NextRequest) {
+  try {
+    const name = request.nextUrl.searchParams.get('name') as string;
+
+    const file = await fs.readFile(process.cwd() + '/app/.well-known/nostr.json/nostr.json', 'utf8');
```

Comment:
Tested against preview deployment at 
`https://bitcoin-dev-project-53brnx5ig-btc-knowledge-base.vercel.app/.well-known/nostr.json?name=_`

got 500 response:
```json
{
  "error": "ENOENT: no such file or directory, open '/var/task/app/.well-known/nostr.json/nostr.json'"
}
``` 
seems it has an error reading that file in prod.
You can try importing it instead.

i.e:
``` ts
import data from './nostr.json'
```